LICENSE.md

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2012-2022 Scott Chacon and others
```

Comment:
I'd also prefer to not have my e-mail listed in the commit title. GitHub will only list e-mails in commits so web-crawlers don't see them.

Reply:
Makes sense, amended the message to just use your name.